apps/mull-ui/src/app/app.tsx

```diff
@@ -50,6 +51,7 @@ export const App = () => {
         <Switch>
           <Route exact path={ROUTES.LOGIN} component={LoginPage} />
           <Route exact path={ROUTES.REGISTER} component={RegisterPage} />
+          <Route exact path={ROUTES.TOKEN_REDIRECT} component={TokenRedirectPage} />
```

Comment:
Just a question, is there a reason why the redirect is a separate page? Would you know of another way to handle this?

Reply:
I decided it would be better to keep it separate from the login page since it has a single purpose of catching a token and logging in the user. However, this could be included in the login component under `/login/<token>` but we would still need to expose a route that handles it.